wordpress_rock/rockcraft.yaml

```diff
@@ -81,8 +81,6 @@ parts:
       mkdir -p wordpress_install_dir
       (cd wordpress_install_dir; $CRAFT_PART_BUILD/wp core download --version=${WP_VERSION} --allow-root)
 
-      rm -rf wordpress_install_dir/wp-content/themes/twentytwentythree
-      rm -rf wordpress_install_dir/wp-content/themes/twentytwentyfour
```

Comment:
Hi, I think there a a few problems with these 2 themes specifically. the `twentytwentythree` theme that comes by default with wordpress seems to be faulty (when you first go to the home page it just shows a blank screen, you'd need to navigate to /wp-admin.php to see the theme automatically uninstalls itself before the homepage can work again).

For the `twentytwentyfour` theme there seems to be a bug in our integration test (specifically this [line](https://github.com/canonical/wordpress-k8s-operator/blob/ab1dbde93509c4a5cd95d4cc65f2b31e399ba415/tests/integration/helper.py#L113) does not work well with the new theme - maybe the API changed?)

Reply:
These 2 lines are workarounds for those problems, by removing `twentytwentythree` and lets wordpress install it during reconciliation. 

for `twentytwentyfour` we'd need to fix the integration test before adding it back in

Reply:
> theme automatically uninstalls

I think this is caused by the theme not in the `_DEFAULT_WORDPRESS_THEMES` list, because the WordPress charm will remove all theme that's not in the `_DEFAULT_WORDPRESS_THEMES` list during the reconciliation, and if the WordPress active theme not in the list, this will cause the WordPress page to go blank.

Reply:
> for twentytwentyfour we'd need to fix the integration test before adding it back in

Yes, I think I have found the reason for the failing, the `twentytwentyfour` theme came with WordPress 6.4 has a redesign of the home page, I have updated the integration test to accommodate that.

Reply:
All the problems in the integration tests have been fixed, I have removed the `test_default_wordpress_themes_and_plugins` test because that test only sees the themes and plugins after the reconciliation which is kind pointless. Sorry I missed your original pull request.